### DISCUSSION TITLE
how to join the server?

### DISCUSSION THREAD

**Discussion by Sneezy123** [LanternPowered/Terre#8] [General]
the game says "Connecting..." constanly when I want to join the "lobby" or "survival"

**Comment by Cybermaxke** [comment]
Did you create and start a Terraria server as well? The supported versions are 1.4.2.3 (desktop) and 1․4․0․5 (mobile), if you want crossplay, use 1․4․0․5 as well. Terre by itself is not a Terraria server, so you need both Terre and at least one Terraria server. Configure the ports accordingly in the config file. A setup overview [here](https://github.com/LanternPowered/Terre/wiki).

**Reply by Cybermaxke** [reply]
Until I finish the wiki, maybe the info/video on my portfolio will also help: https://www.seppevolkaerts.be/

**Comment by Sneezy123** [comment]
Doesn't work

**Reply by Sneezy123** [reply]
Where should you change the ip in the config.yaml
